forge-gui/res/cardsfolder/upcoming/you_cannot_hide_from_me.txt

```diff
@@ -0,0 +1,12 @@
+Name:You Cannot Hide from Me
+ManaCost:no cost
+Types:Ongoing Scheme
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Command | Execute$ TrigPump | TriggerDescription$ At the beginning of combat on your turn, up to one target creature gets +2/+2 and gains vigilance until end of turn. It can't be blocked this turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature | NumAtt$ 2 | NumDef$ 2 | KW$ Vigilance | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ExileOnMoved$ Battlefield | RememberObjects$ ParentTarget | StaticAbilities$ Unblockable
+SVar:Unblockable:Mode$ CantBlockBy | ValidAttacker$ Card.IsRemembered | Description$ This creature can't be blocked this turn.
+T:Mode$ Phase | Phase$ End Of Turn | ValidPlayer$ Opponent | Execute$ Abandon | TriggerZones$ Command | CheckSVar$ X | SVarCompare$ LTY | TriggerDescription$ At the beginning of each opponent's end step, if that player has less than half their starting life total, abandon this scheme.
+SVar:Abandon:DB$ Abandon
+SVar:X:TriggeredPlayer$LifeTotal

```

Comment:
imo this seems untested because you're trying to use triggeringsobjects while checking the condition

Reply:
The schemes were generally so straining that I had no energy left for testing :) Their effects are usually novel and complicated.

Reply:
uhh...do you mean it's better if I fix it?

Reply:
Probably, I'm not sure how :(

Reply:
I think you could just use PlayerCountDefinedActivePlayer$ so it's not reliant on the trigger

Reply:
OK, and in this case, even theoretical later implementation of shared turns wouldn't break it.